eagleproject/core/views.py

```diff
@@ -285,3 +287,26 @@ def _get_trailing_apy():
     return 0 # Temporary
     apy = ((1 + apr / periods_per_year / 100) ** periods_per_year - 1) * 100
     return round(apy, 2)
+
+def staking_stats(request):
+    with connection.cursor() as cursor:
+        query = """
+        select count(*) as count, sum(staked_amount) as total_staked from (
+            select user_address, sum(
+                case 
+                when is_staked then amount
+                else amount * -1
+                end
+            ) as staked_amount from core_ognstaked group by user_address
```

Comment:
Not sure about this pattern.  Anyone who's withdrawn their stakes will show as negative staked_amounts.  Maybe doesn't need to be dealt with now if it works for you for the dapp.

Reply:
I'm taking a sum and ignoring rows where sum <= 0. So, yeah, it works. Will see if there is a better way to do it.